README.md

```diff
@@ -248,3 +248,35 @@ PIXL data extracts include the below assumptions
 
 - (MRN, Accession number) is unique identifier for a report/DICOM study pair
 - Patients have a single _relevant_ MRN
+
+
+
+## File journey overview
+Files that are present at each step of the pipeline.
+
+### Resources in source repo (for test only)
+```
+test/resources/omop/public /*.parquet
+....................private/*.parquet
+....................extract_summary.json
```

Comment:
might be nicer in a tree format but not strongly held

Reply:
This is so much easier to edit though!